samples/ts/21.react-media-template/src/live-share-hooks/index.ts

```diff
@@ -0,0 +1,17 @@
+import { useSharedObjects } from "./useSharedObjects";
```

Comment:
FYI, learned via Steve via Ryan:
```suggestion
export * from "./useSharedObjects";
```

works, meaning you don't need to have the export object at the bottom